### DISCUSSION TITLE
Problems installing antismash on server

### DISCUSSION THREAD

**Discussion by SevanEsaian** [antismash/antismash#458] [Q&A]
I am trying to install antismash on the university server. Based on the attempts I have tried, the installation either gets stuck while solving the environment or gives me an ImportError. 

When I run the following:
`conda activate antismash`
`conda install antismash`
It gets stuck solving the environment. 

When I run:
`conda activate antismash`
`conda install antismash biopython=1.76 python=3.7`
It gives me the following ImportError:
`ImportError: cannot import name 'Markup' from 'jinja2'.`

Do you have any suggestions for how I can resolve the ImportError or suggest an alternative installation method?

**Comment by SJShaw** [comment]
With 6.1 that's been fixed, but #437 has a way to work around it if conda still hasn't updated the version they're hosting.

**Reply by SevanEsaian** [reply]
Thank you for the feedback. I will try the solutions from #437  and will let you know if I am successful. Very much appreciated.
